ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -35,6 +35,8 @@ using namespace skia::textlayout;
 std::queue<rnsKey> inputQueue;
 sem_t jsUpdateMutex;
 std::mutex privateVarProtectorMutex;
+static bool previousKeyRepeat;
+unsigned int queueIndex;
```

Comment:
can logic to reset these global ( queueIndex & PreviosKeyRepeat) be added in constructor as it not a class member.

Reply:
Address